docs/join-the-network/run-avail/other-nodes/0050-avail-bootstrap-node.md

```diff
@@ -0,0 +1,60 @@
+---
+id: avail-bootstrap-node
+title: Run a Bootstrap Node
+sidebar_label: Run a Bootstrap Node
+description: Learn how to run a boostrap node
+keywords:
+  - docs
+  - avail
+  - node
+  - data availability
+  - da
+image: https://availproject.github.io/img/avail/AvailDocs.png
+slug: avail-bootstrap-node
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Bootstrap Node Setup
```

Comment:
We need to clarify that this is for LC and not the Substrate node - it might create confusion.

Reply:
Agree with @prabal-banerjee perhaps something like this 

`bootstrap nodes play a crucial role in facilitating initial network discovery and connection establishment. By connecting to the bootstrap nodes, new nodes can discover and connect to other peers in the network, gradually expanding the network's connectivity`